content/enterprise_influxdb/v1.9/guides/fine-grained-authorization.md

```diff
@@ -48,8 +48,18 @@ In a typical cluster configuration, the HTTP ports for data nodes
 You may need to work with your network administrator to gain access to the meta node HTTP ports.
     {{% /note %}}
 
-4. _(Optional)_ [Create roles](#manage-roles).
-   Roles let you grant permissions to groups of users assigned to each role.
+4. [Create roles](#manage-roles). Roles let you grant permissions to groups of users assigned to each role.
+   As administrator, create a user with a standard grant, as shown in the example below, and then restrict by role or via grants. 
+   You **must** set up coarse read/write permissions **FIRST**, then restrict to a fine grain.
```

Comment:
```suggestion
4. [Create roles](#manage-roles) to grant permissions to users assigned to a role. Do the following:
   a.  As administrator, grant users all permissions. The example below grants users `east` and `west` all permissions on the `datacenters` database.  
   b. Add a fine grain permission to users as needed.
```

Reply:
- Does this example create a role...or show how to create a user and grant all permissions (before the role is created)? Let's clarify accordingly.
- Let's add an example of how to specify a fine grain permission (to either user) here, clarifying in text and example.